test/test_unary_ufuncs.py

```diff
@@ -568,7 +568,7 @@ def test_digamma(self, device, dtype):
         # TODO: Add value `-1931.99999994`, to the tensor below when
```

Comment:
Since this is the issue this PR is fixing, this note should be removed, too

Reply:
done :)